include/remill/Arch/Runtime/Intrinsics.h

```diff
@@ -137,8 +137,8 @@ __remill_flag_computation_carry(bool result, ...);
 [[gnu::used]] extern Memory *__remill_async_hyper_call(State &, addr_t ret_addr,
                                                        Memory *);
 
-[[gnu::used]] extern Memory *__remill_sync_hyper_call(State &, Memory *,
-                                                      SyncHyperCall::Name);
+[[gnu::used, gnu::always_inline]] extern Memory *
```

Comment:
This tag is important. If we don't put it here, the runtime will implicitly have this function marked with `noinline` as a LLVM function attribute. The function will not get inlined and then when we move the functions across to the target module, we will not copy across the `__remill_sync_hyper_call` definition and it will be lost.

Reply:
That is.. subtle. Can you add it as a comment in the code?

Reply:
[0574f2a](https://github.com/lifting-bits/remill/pull/611/commits/0574f2a4e1994026c8bf3ce58241ff360e19652c)